package.json

```diff
@@ -82,12 +82,14 @@
     "@types/storybook__addon-actions": "3.4.1",
     "@types/storybook__addon-info": "3.4.2",
     "@types/storybook__addon-knobs": "3.4.1",
+    "cache-loader": "^2.0.1",
```

Comment:
let's pin version for both packages please

Reply:
✅